docs/blog/2019-02-20-getting-started-with-gatsby-themes/index.md

```diff
@@ -0,0 +1,266 @@
+
+```
+mkdir site
+mkdir packages
+cd packages
+mkdir theme
+cd theme
+yarn init
+cd ../../site/
+yarn init
+```
+
+After using ```yarn init``` you will be asked a few questions. The main thing you will want to pay attention to is the entry point (index.js). For the *theme* directory, you can leave the entry point as *index.js* (just make sure you have an *index.js* file) and for the *site* directory, you can make entry point *gatsby-config.js*. Next, you will want to go into your *site* directory and do ```yarn workspace site add gatsby```.
+
+Hooray! Gatsby should now be added to your site directory if you look in your *package.json* file. Adjustments you make to your file is adding "scripts" and adding the name of your Gatsby theme, in this case, *theme*, to your dependencies.
+
+```
+{
+"name": "site",
+"version": "0.0.1",
+"description": "A site used to play with Gatsby themes",
+"main": "gatsby-config.js",
+"license": "MIT",
+"scripts":{
+"develop": "gatsby develop",
+"build": "gatsby build"
+    },
+"dependencies": {
+"gatsby": "^2.0.118",
+"theme":"*"
+    }
+}
+```
+
+You will want to add Gatsby, React, and ReactDOM to as dev dependencies for *site*.
+
+```yarn workspace site add gatsby react react-dom -D```
+
+Then you will navigate out of the *site* directory and add Gatsby, React, and ReactDOM as dev dependencies for *theme*.
+
+```yarn workspace theme add gatsby react react-dom -D```
+
+You will want to make Gatsby, React, and ReactDom peer dependencies in both the *site* and *theme* directories.
+
+```
+"devDependencies": {
+"gatsby": "^2.0.118",
+"react": "^16.8.1",
+"react-dom": "^16.8.1"
+},
+"peerDependencies": {
+"gatsby": "^2.0.118",
+"react": "^16.8.1",
+"react-dom": "^16.8.1"
+}
+```
+
+# Installing MDX and gatsby-plugin-page-creator
+### What is MDX?
+> MDX is markdown for the component era. It lets you write JSX embedded inside markdown. That's a great combination because it allows you to use markdown's often terse syntax (such as # heading) for the little things and JSX for more advanced components.
+
+Read more about Gatsby+MDX [here.](https://gatsby-mdx.netlify.com/)
+
+In your *theme* directory, add src/pages/index.mdx
+
+Then you need to add gatsby-mdx and MDX as dependencies.
+
+```yarn workspace theme add gatsby-mdx @mdx-js/mdx @mdx-js/tag```
+
+Next, you will want to add gatsby-plugin-page-creator
+
+```yarn workspace theme add gatsby-plugin-page-creator```
+
+> Gatsby plugin that automatically creates pages from React components in specified directories. Gatsby includes this plugin automatically in all sites for creating pages from components in src/pages. With this plugin, any file that lives in the src/pages folder (or subfolders) will be expected to export a React Component to generate a Page.
+
+Read more about the page creator plugin [here.](/packages/gatsby-plugin-page-creator/)
```

Comment:
"In the future, Gatsby will automatically handle adding the page-creator plugin to themes as well" would be a good note to add here.

Reply:
Got it!